eve/workers/pod-builder/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos/7
+
+COPY requirements-eve.txt /tmp/
+
+RUN yum install -y \
+    python2-pip \
+    genisoimage \
+    git \
+    && curl -fsSL https://get.docker.com/ | sh
+    && adduser --home /home/eve eve \
+    && usermod -aG docker eve
+    && pip2 install -r /tmp/requirements-eve.txt \
+    && pip2 install buildbot-worker==0.9.0.post1
+
+USER eve
+WORKDIR /home/eve/workspace
+ENV DOCKER_HOST=localhost
```

Comment:
IMHO you can remove this line as well. Per example if you're working with this image on your laptop you might bind the docker socket instead of using localhost